src/web_interface/components/jinja_filter.py

```diff
@@ -125,7 +125,7 @@ def _setup_filters(self):
         self._app.jinja_env.filters['fix_cwe'] = fix_cwe
         self._app.jinja_env.filters['vulnerability_class'] = vulnerability_class
         self._app.jinja_env.filters['auth_enabled'] = self.check_auth
-	self._app.jinja_env.filters['hash'] = lambda s: hashlib.md5(s.encode()).hexdigest()
+        self._app.jinja_env.filters['md5_hash'] = lambda s: get_md5(s.encode())
```

Comment:
The helper function should be capable of handling any kind of input. Is the encoding really necessary at this point?